README.md

```diff
@@ -1,7 +1,7 @@
 SonarQube/SonarCloud Azure Pipelines Extension
 ========================================
 
-[![Build Status](https://dev.azure.com/sonarsource/_apis/public/build/definitions/399fb241-ecc7-4802-8697-dcdd01fbb832/10/badge)](https://dev.azure.com/sonarsource/DotNetTeam%20Project/_build/index?definitionId=10)
+[![Build Status](https://dev.azure.com/sonarsource/DotNetTeam%20Project/_apis/build/status%2FSonarSource.sonar-scanner-vsts?branchName=master)](https://dev.azure.com/sonarsource/DotNetTeam%20Project/_build/latest?definitionId=126&branchName=master)
```

Comment:
To clarify, definitionId is the id of build pipeline, so this is a static value ?

Reply:
Yes this is a static value, `definitionId` is the pipeline ID, you can get it in the permalink to access a specific pipeline